TF migration docs - round 6 (#2045)
saucelabs/sauce-docs: docs/test-fairy/integrations/slack.md

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

> ## One-click integration

Author: ```suggestion
## One-click Integration
```

> ## One-click Integration

Author: ```suggestion
## One-Click Integration
```

TestFairy integrates with Slack seamlessly, providing human-readable, real-time notifications for your selected events.

1. Head over to your [TestFairy Dashboard](https://app.testfairy.com), and select **Account Preferences**.
   <img src={useBaseUrl('/img/test-fairy/integrations/slack/preferences-link.png')} alt="account preferences"/>

1. Select **Integrations**, then click **Add integration**.
   <img src={useBaseUrl('/img/test-fairy/integrations/slack/account-settings-4.png')} alt="add integration"/>

1. Enter the domain of your Slack account:
   <img src={useBaseUrl('/img/test-fairy/integrations/slack/slack-domain-1.png')} alt="slack domain"/>

> 1. This determines where your notifications will appear. You may select an existing channel or create a new channel. Choose your channel and select **Authorize**.

Author: ```suggestion
1. It determines where your notifications appear. You may select an existing channel or create a new channel. Choose your channel and select **Authorize**.
```

   <img src={useBaseUrl('/img/test-fairy/integrations/slack/slack-1c.png')} alt="authorize channel"/>

> 1. You will be returned to your TestFairy page with the `URL` and `Events` already filled in and selected.

Author: ```suggestion
1. You are returned to your TestFairy page with the `URL` and `Events` already filled in and selected.
```
